results-loader/src/utils/results-utils.ts

```diff
@@ -43,3 +44,28 @@ export const getFirebaseJwt = (portalUrl: string, portalAccessToken: string, fir
     .then(response => response.json())
     .then(json => json.token);
 };
+
+interface IGetCredentials {
+  resource: Resource;
+  firebaseJwt?: string;
+  tokenServiceEnv: EnvironmentName;
+}
+export const getCredentials = async ({resource, firebaseJwt,
+  tokenServiceEnv}: IGetCredentials): Promise<Credentials> => {
+
+  // The jwt will be ignored if there is a readWriteToken on the resource
+  const client = new TokenServiceClient({ env: tokenServiceEnv, jwt: firebaseJwt });
+  const readWriteToken = client.getReadWriteToken(resource) || "";
+
+  return readWriteToken ? await client.getCredentials(resource.id, readWriteToken) :  await client.getCredentials(resource.id);
+};
+
+export const listResources = async (firebaseJwt: string, amOwner: boolean,
+  tokenServiceEnv: EnvironmentName, resourceType: ResourceType) => {
+  const client = new TokenServiceClient({ jwt: firebaseJwt, env: tokenServiceEnv });
+  return client.listResources({
+    type: resourceType,
+    tool: "example-app", // TODO: this needs to be set to valid string based on the app that created the resource
```

Comment:
Right. Probably something like `"athena-reports"`.  But it is fine to leave this "example-app" for now.
When we switch to `"athena-reports"` we'll need to add a document in the token-service firestore database for this new tool.

Reply:
Agreed.  In general we will need to lock down the results loader name in a handful of places and it will probably makes things easier if we are consistent (this tool name as well as places like the deployed URL, the oauth client name, etc.).  Then again, maybe it doesn't matter if they differ...